modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Sorting.tsx

```diff
@@ -228,17 +228,17 @@ const AddFDSSortModalContent = ({
 				last={
 					<ClayButton.Group spaced>
 						<ClayButton
-							displayType="secondary"
-							onClick={() => closeModal()}
+							disabled={isSubmitting || !selectedField}
```

Comment:
Changes about the order of buttons are better in a separate commit, no need to create a separate task as we're dealing with these buttons already in this PR.